lib/movable_ink/version.rb

```diff
@@ -1,5 +1,5 @@
 module MovableInk
   class AWS
-    VERSION = '2.6.5'
+    VERSION = '2.7.0'
```

Comment:
Please change it in Gemfile.lock file too:

https://github.com/movableink/awslib/blob/master/Gemfile.lock#L4